gammapy/time/tests/test_lombscargle_gammapy.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+# import sys
+# sys.path.insert(0, '/afs/ifh.de/group/amanda/scratch/wegenmat/time-series-analysis/time-series-analysis')
```

Comment:
Again: remove commented out code here and change filename to `test_lombscargle.py`.